packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -92,7 +92,7 @@ export class SidebarComponent {
 			// Explicitly select only the child headings and combine with the text matching engine.
 			// This works better than using CSS pseudo-classes like `:has-text` or `:text-matches` for text
 			// matching.
-			selector = `${ selectors.subheading } >> text="${ subitem }"`;
+			selector = `${ selectors.subheading } :text("${ subitem }"):visible`;
```

Comment:
This is required change; otherwise, hidden subheadings are also matched.

Reply:
I think an exact text match via `:text-is()` would be the safer choice here?